specification/nodes/_types/Stats.ts

```diff
@@ -160,58 +382,157 @@ export class KeyedProcessor {
 }
 
 export class Processor {
+  /**
+   * Number of documents transformed by the processor.
+   */
   count?: long
+  /**
+   * Number of documents currently being transformed by the processor.
+   */
   current?: long
+  /**
+   * Number of failed operations for the processor.
+   */
   failed?: long
+  /**
+   * Time, in milliseconds, spent by the processor transforming documents.
+   */
   time_in_millis?: DurationValue<UnitMillis>
 }
 
 export class AdaptiveSelection {
+  /**
+   * The exponentially weighted moving average queue size of search requests on the keyed node.
+   */
   avg_queue_size?: long
+  /**
+   * The exponentially weighted moving average response time of search requests on the keyed node.
+   */
   avg_response_time?: Duration
+  /**
+   * The exponentially weighted moving average response time, in nanoseconds, of search requests on the keyed node.
+   */
   avg_response_time_ns?: long
+  /**
+   * The exponentially weighted moving average service time of search requests on the keyed node.
+   */
   avg_service_time?: Duration
+  /**
+   * The exponentially weighted moving average service time, in nanoseconds, of search requests on the keyed node.
+   */
   avg_service_time_ns?: long
+  /**
+   * The number of outstanding search requests from the node these stats are for to the keyed node.
```

Comment:
```suggestion
   * The number of outstanding search requests to the keyed node from the node these stats are for.
```

Reply:
I had to read this sentence five times to understand what it says. Maybe this makes it easier to read?

Reply:
Great suggestion, thank you!